.github/workflows/tests.yml

```
@@ -594,9 +594,17 @@ jobs:
                 run: uv run --frozen --no-build python -m scripts.engineering.qa check-docs-drift
 
             -   name: Validate docker-compose.yml syntax
+                env:
+                    # Placeholders only for `docker compose config` interpolation.
+                    # Runtime stacks still require real secrets from operator env.
+                    NEO4J_PASSWORD: ci-compose-config-validation
                 run: docker compose config > /dev/null
 
             -   name: Validate docker-compose.monitoring.yml syntax
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)